src/electionguard/encrypt.cpp

```diff
@@ -135,7 +135,7 @@ namespace electionguard
     }
 
     unique_ptr<CiphertextBallot>
-    EncryptionMediator::encrypt(const PlaintextBallot &ballot, bool shouldVerifyProofs /* = true */,
+    EncryptionMediator::encrypt(const PlaintextBallot &ballot, bool verifyProofs /* = true */,
```

Comment:
no actual logic changed in this class, just moving variables around to make the internals easier to work with